src/app/public/modules/toggle-switch/toggle-switch.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+$indicator-size: 22px;
+
+.sky-toggle-switch {
+  display: inline-flex;
+  align-items: baseline;
+}
+
+.sky-toggle-switch-button {
+  border: 1px solid $sky-border-color-neutral-medium;
+  background-color: $sky-background-color-neutral-light;
+  padding: 1px;
+  border-radius: $indicator-size + 2;
+  width: $indicator-size * 2 + 2;
+  line-height: $sky-line-height-base;
+
+  &:hover:not(.sky-toggle-switch-readonly) {
+    border-color: $sky-highlight-color-info;
+    border-width: 2px;
+    padding: 0;
+    cursor: pointer;
+  }
+
+  &.sky-toggle-switch-checked {
+    background-color: $sky-background-color-info-light;
+    border-color: $sky-highlight-color-info;
+    border-width: 2px;
+    padding: 0;
+
+    .sky-toggle-switch-indicator {
+      left: 100%;
+    }
+  }
+
+  &.sky-toggle-switch-readonly {
+    background-color: $sky-background-color-disabled;
+    border-color: transparent;
+
+    .sky-toggle-switch-indicator {
+      background-color: $sky-background-color-neutral-light;
+    }
+  }
+
+  & + .sky-toggle-switch-label {
+    margin-left: $sky-margin;
+  }
+}
+
+.sky-toggle-switch-indicator {
+  height: $indicator-size;
+  width: $indicator-size;
+  display: block;
+  position: relative;
+  top: 0;
+  left: $indicator-size;
+  margin-left: -$indicator-size;
+  border-radius: 100%;
+  box-shadow: 0px 1px 2px 0px rgba(0, 0, 0, 0.75);
+  background-color: $sky-color-white;
+  transition: left 150ms;
```

Comment:
This transition is also happening when the control first loads (Firefox). Should we prevent that from happening?

Reply:
Yep! Done.